Maths/test/QuadraticFormula.test.js

```diff
@@ -1,21 +1,8 @@
-import { quadraticFormula } from '../QuadraticFormula'
+import { solveQuadraticEquation } from '../QuadraticFormula'
 
-test('Testing on quadraticFormula', () => {
-  const quadraticForm = quadraticFormula(1, -3, -4)
-  expect(quadraticForm).toBe('the roots are X1=4 and X2=-1')
-})
-
-test('Testing on quadraticFormula', () => {
-  const quadraticForm = quadraticFormula(1, 5, 6)
-  expect(quadraticForm).toBe('the roots are X1=-2 and X2=-3')
-})
-
-test('Testing on quadraticFormula', () => {
-  const quadraticForm = quadraticFormula(1, -3, 8)
-  expect(quadraticForm).toBe('the roots do not exist or the roots are imaginary')
-})
-
-test('Testing on quadraticFormula', () => {
-  const quadraticForm = quadraticFormula(1, -2, 9)
-  expect(quadraticForm).toBe('the roots do not exist or the roots are imaginary')
+test('Quadratic Equation', () => {
+  expect(solveQuadraticEquation(1, -3, -4)).toStrictEqual([4, -1])
+  expect(solveQuadraticEquation(1, 5, 6)).toStrictEqual([-2, -3])
+  expect(solveQuadraticEquation(1, -3, 8)).toStrictEqual([])
```

Comment:
Include a test case where there is only one solution please.

Reply:
Ok, perfect, I think I did it 🤞